dbt/src/main/dbt_snowflake_project/profiles.yml

```diff
@@ -0,0 +1,21 @@
+dbt_snowflake_project:
+  target: dev
+  outputs:
+    dev:
+      type: snowflake
+      account: "SPMXIFV-MV79218"
```

Comment:
This should probably be put in the `.envrc` too.

Reply:
I will, though it's not as sensitive as the user/pass. Thanks